### DISCUSSION TITLE
What is the different between website and a blogsite?

### DISCUSSION THREAD

**Discussion by BogusCurry** [datenstrom/community#194] [Ask a question] [answered]
Hello ;D
Yellow is a nice try for a CMS, thanks for that ;D
My question is .. What is the different between website and a blogsite ?

Greetings
Bogus

**Comment by schulle4u** [comment, marked as answer]
A website mostly consists of static pages, meaning the content is not frequently being updated. A blogsite, also called blog or weblog, is a website which contains a diary-like content, meaning it is organized and browseable by date, category or tag. [Wikipedia has closer explanation](https://en.wikipedia.org/wiki/Blog). 

HTH
Steffen
